### DISCUSSION TITLE
How to suppress/hide info?

### DISCUSSION THREAD

**Discussion by noobistz** [explosion/spaCy#11480] [Help: Other Questions] [answered]
Sometimes the following info will appear when attempting to predict if a given string contains any relations using a relation extraction model: 
"could not determine any instance in the doc - ..." 

How can i go about suppressing/hiding this info msg?

**Comment by richardpaulhudson** [comment]
Could you please add a bit more information about exactly what you're doing and with which components, preferably including a code snippet?

**Reply by noobistz** [reply]
I am just using the relation extraction model from https://github.com/explosion/projects/tree/v3/tutorials/rel_component to predict on a given string, that may or may not contain a viable relation. 

"Could not determine any instances in the doc - returning doc as is." will understandably show up if there are no viable relations in the given string, but is there a way to suppress/hide this info?

**Comment by richardpaulhudson** [comment, marked as answer]
This warning actually comes from the code you're running within the project directory, so you can just remove the line of code that prints it: https://github.com/explosion/projects/blob/24553c6786aeaa44d5f308be8a43504aec073c87/tutorials/rel_component/scripts/rel_pipe.py#L77.

**Reply by noobistz** [reply]
thanks for your help!
